src/routes/routes.tsx

```diff
@@ -45,13 +46,16 @@ export const AppRoutes = () => {
         <Route path="give" element={<AllocationPage />} />
         <Route path="map" element={<LazyAssetMapPage />} />
         <Route path="vouching" element={<VouchingPage />} />
-        <Route path="admin" element={<AdminRouteHandler />}>
+        <Route path="members" element={<AdminRouteHandler />}>
           <Route path="" element={<AdminPage />} />
           <Route
```

Comment:
This connect-integration route should be moved down so it is still `/circle/:id/admin/connect-integration`

Reply:
done